### PR TITLE
Fearture/x nwb dat converter one file per group

### PR DESCRIPTION
**Pull request by t-b**

### CODE REVIEW

File: ipfx/bin/run_x_to_nwb_conversion.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from ipfx.x_to_nwb.DatConverter import DatConverter
 
 
-def convert(inFileOrFolder, overwrite=False, fileType=None, outputMetadata=False, outputFeedbackChannel=False):
+def convert(inFileOrFolder, overwrite=False, fileType=None, outputMetadata=False, outputFeedbackChannel=False, oneFilePerGroup=False):
```
Comment (dyf) [inline ipfx/bin/run_x_to_nwb_conversion.py]: Can you default this to True?

Comment (t-b) [reply ipfx/bin/run_x_to_nwb_conversion.py]: @dyf Done.

**Comment by t-b** [comment]
@dyf Done, now also in the main conversion script. I've renamed the flag to multipleGroupsPerFile and that now always defaults to False.

**Comment by dyf** [comment]
Thanks @t-b.  Just to make sure -- the default behavior is that each NWB is all recordings from single cell now, right?

**Comment by t-b** [comment]
@dyf Yes, sorry for the boolean confusion.

**Pull request merged**